### DISCUSSION TITLE
[ FF7 ] Game Crashes Upon Using Cait Sith's Slots Limit Break w/ Lucky Cait Sith Gameplay Tweak Enabled

### DISCUSSION THREAD

**Discussion by gs82139** [julianxhokaxhiu/FFNx#233] [Q&A] [answered]
[FFNx.log](https://github.com/julianxhokaxhiu/FFNx/files/6417235/FFNx.log)

<img width="1280" alt="FF7 Crash" src="https://user-images.githubusercontent.com/67979853/116924463-e757f200-ac25-11eb-89f7-0c3288a34d28.PNG">

Just like the title suggests, I'm having an issue with the "Lucky Cait Sith" Gameplay Tweak using the Gameplay Mod found in 7th Heaven from qhimm. I'm using 7th Heaven 2.2.3.522 for reference. I also went back to test the latest version of 2.0 (2.0.3.406) and it still happens there as well.

I can use any of the other Gameplay Tweak options in the mod, however, any time I use Cait Sith's Slots limit break, the game crashes with the dialog box found in the screenshot. I have tested this in multiple battles and it happens every time no matter what battle I'm in or where it takes place. 

I have further isolated the issue as I disabled every other mod in my install, and even disabled every other aspect of the Gameplay Mod EXCEPT the Lucky Cait Sith tweak and the issue still occurs, I'm pretty sure it isn't because of a mod conflict because of this.

Any suggestions on how to fix the issue?

**Comment by julianxhokaxhiu** [comment, marked as answer]
Hi,

Thanks for having me approached but unfortunately the crash message is misleading. When you're using 7th Heaven and Mods, you should approach the relative authors and support channels, not me directly. I'm sorry, can't help directly as the crash is not related to a vanilla gameplay session.

Cheers!

**Comment by gs82139** [comment]
Got it, I'll try posting the discussion topic to Qhimm itself and hopefully get the issue resolved. Thanks for your help!

Get Outlook for Android<https://aka.ms/ghei36>

________________________________
